vaccine_feed_ingest/runners/mn/gov/normalize.py

```diff
@@ -44,29 +45,34 @@ def _get_id(site: dict) -> str:
     return f"{runner}_{site_name}:{arcgis}_{layer}_{data_id}"
 
 
+def _cleanup_url(url):
+    if not url:
+        return None
+    # if "@" in url:
```

Comment:
Why did you comment out the email check?

Reply:
i essentially removed this whole fixup function by comments in an attempt to convert it to the utility URL normalizer, and then i selectively re-added the checks (and added more checks) to fix the issues that the util function doesnt handle.

in effect this is also partly a migration to using the common url normalizer

Reply:
Ah, I see, then just remove this commented out code because it isn't relevant here anymore.